lib/iris/tests/integration/test_pp.py

```diff
@@ -587,5 +588,56 @@ def test_as_pairs(self):
             self.assertEqual(cube.attributes['pseudo level'], 3)
 
 
+class TestSaveLBPROC(tests.IrisTest):
+
+    def create_cube(self):
+        cube = Cube(np.zeros((2, 3, 4)))
+        cube.add_dim_coord(
+            DimCoord(standard_name='time',
+                     units=Unit('days since 0001-01-01 00:00:00',
+                                calendar='gregorian'),
+                     points=np.arange(2),),
+            0)
+        cube.add_dim_coord(DimCoord(standard_name='longitude',
+                                    units='degrees',
+                                    points=np.arange(3),), 1)
```

Comment:
I just feel on principal that your choice of implementation (i.e. choosing `var_name` or `standard_name` which are functionally different in the wider context) should not be dictated by the coding standards we choose to enforce. You chose `standard_name` initially, presumably because that is the implementation you really wanted, and you shouldn't have to change it to conform to how the code should look. Coding standards are important, but there are better ways of improving to conform.